devenv/docker/blocks/openldap/prepopulate/2_users.ldif

```diff
@@ -78,3 +78,31 @@ objectClass: inetOrgPerson
 objectClass: organizationalPerson
 sn: ldap-torkel
 cn: ldap-torkel
+
+# admin for posix group (without support for memberOf attribute)
```

Comment:
I took these people from my imagination, hopefully it's close to the real setups